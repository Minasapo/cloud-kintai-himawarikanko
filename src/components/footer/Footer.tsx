import { Box, Typography } from "@mui/material";

export default function Footer() {
  return (
    <footer
      style={{
        backgroundColor: "#0FA85E",
      }}
    >
      <Box textAlign="center" sx={{ p: 1 }}>
        <Typography
          sx={{
            variant: "body2",
            color: "white",
          }}
        >
          Â© 2025 Virtual Tech Japan Inc.
        </Typography>
      </Box>
    </footer>
  );
}
