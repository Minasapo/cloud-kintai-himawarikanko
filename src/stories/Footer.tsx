import { Box, Typography } from "@mui/material";

const Footer = () => (
  <footer>
    <Box bgcolor="#0FA85E" textAlign="center" sx={{ p: 1 }}>
      <Typography variant="body1" color="white">
        Â© 2022 Virtual VirtualTech Japan Inc. All rights reserved.
      </Typography>
    </Box>
  </footer>
);

export default Footer;
