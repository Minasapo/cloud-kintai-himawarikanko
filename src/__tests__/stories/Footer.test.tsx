import { render } from "@testing-library/react";
import Footer from "../../stories/Footer";

describe("Footer", () => {
  test("Default", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
