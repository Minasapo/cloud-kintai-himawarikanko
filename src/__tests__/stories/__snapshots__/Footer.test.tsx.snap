// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <footer>
    <div
      class="MuiBox-root css-1e47hhr"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-jb1ygz-MuiTypography-root"
      >
        Â© 2022 Virtual VirtualTech Japan Inc. All rights reserved.
      </p>
    </div>
  </footer>
</div>
`;
